### ISSUE TITLE
Notification settings checkboxes are not accessible

### ISSUE THREAD

**Issue by ndarilek** [vector-im/element-web#24416]
### Steps to reproduce

1. Launch a screen reader on your platform of choice--any should do.
2. Right-click a room in the rooms tree.
3. Click Settings.
4. Click Notifications.

#### What did you expect?

"Default", "All messages", etc. should be presented to my screen reader as a radio button with only one checked.

#### What happened instead?

Only the text of the options is spoken, with no indication that they are clickable or of which one is selected.

### Operating system

Windows

### Application version

All recent versions

### How did you install the app?

Desktop app

### Homeserver

Synapse, latest

### Will you send logs?

No

**Comment by daniellekirkwood** [comment]
We also have a plan (in the next few weeks) to change these screen slightly so let's keep that in mind: https://github.com/vector-im/element-web/issues/24291

**Comment by t3chguy** [comment]
It seems that this radio group is using arrow keys for navigation/selection @ndarilek - though I am having it correctly read to me in Firefox + Voiceover the currently selected option.

> Default Get notifications as set up in your settings, selected, radio button, 1 of 4.

Semantically it is based on https://w3c.github.io/aria-practices/examples/radio/radio.html

Does that example behave better for you @ndarilek?

**Comment by ndarilek** [comment]
OK, interesting. Using Firefox under Windows with NVDA. It works in 
focus mode, but not in browse mode.


The linked example works both in focus mode and browse mode.

**Comment by t3chguy** [comment]
Strange, structurally we should be valid. Input inside a label with additional text content (Default). Input has an aria-describedby linking to a span with the referenced ID. Will try to test in NVDA shortly.

**Comment by t3chguy** [comment]
So I could reproduce weirdness in Browse mode with it just saying `Blank` - but only if I tab into it in browse mode, when using arrows to navigate browse mode with enter to confirm the UX works exactly as I expected, reading each option fully and enter to confirm
